src/instructlab/model/evaluate.py

```diff
@@ -314,6 +325,11 @@ def launch_server(
     type=click.Path(),
     help="Path where all the MMLU Branch tasks are stored. Needed for running mmlu_branch.",
 )
+@click.option(
+    "--gpus",
+    type=click.INT,
+    help="Number of GPUs to utilize for evaluation (not applicable to llama-cpp)",
```

Comment:
I still don't understand why we are starting to have backend-specific flags. I mean `--gpus` seems quite generic.

Reply:
The goal isn't to be backend specific.  The flag should apply to any backend that needs the number of gpus specified in order to consume them.

Reply:
I guess I misunderstood the intention, thanks for clarifying!